extension/js/background.js

```diff
@@ -42,58 +51,88 @@ function acceptInput(text, disposition) {
         return;
     }
     switch (disposition) {
-    case "currentTab":
-        chrome.tabs.update({url: text});
-        break;
-    case "newForegroundTab":
-        chrome.tabs.create({url: text});
-        break;
-    case "newBackgroundTab":
-        chrome.tabs.create({url: text, active: false});
-        break;
+        case "currentTab":
+            chrome.tabs.update({
+                url: text
+            });
+            break;
+        case "newForegroundTab":
+            chrome.tabs.create({
+                url: text
+            });
+            break;
+        case "newBackgroundTab":
+            chrome.tabs.create({
+                url: text,
+                active: false
+            });
+            break;
     }
 }
 
 function init() {
 
-    /*db = new PouchDB('main')
-    //db.plugin()
-    var remoteCouch = false;*/
-
+    db = new PouchDB('main')
+        // db.plugin()
+    var remoteCouch = false;
+    db.search({
+        fields: ['text'],
+        build: true
+    }).then(function (info) {
+        // if build was successful, info is {"ok": true}
+    }).catch(function (err) {
+        // handle error
+        console.log(err);
+    });
     window.preloaded = [];
     window.cache = {};
-    chrome.storage.local.get(['blacklist', 'preferences'], function(items) {
+    chrome.storage.local.get(['blacklist', 'preferences'], function (items) {
         var obj = items['blacklist'];
         if (obj === undefined || !('PAGE' in obj && 'SITE' in obj && 'REGEX' in obj)) {
-            window.blacklist = {'PAGE':[], 'REGEX':[], 'SITE':[]}; // show example in page
-            chrome.storage.local.set({'blacklist':blacklist});
+            window.blacklist = {
+                'PAGE': [],
```

Comment:
currently blacklist has page, regex, site. In the future you might add a 4th thing, for example "youtubeChannels" (random example). With the current check you either have all of them or none of them, it would be better to check them individually